deploy/helm/elastic-agent/templates/integrations/_kubernetes/_kubernetes_logs_containers.tpl

```diff
@@ -17,12 +17,19 @@ Config input for container logs
     namespace: {{ .Values.kubernetes.namespace }}
   use_output: {{ .Values.kubernetes.output }}
   streams:
+  {{- if $.Values.kubernetes.containers.logs.rotated_logs }}
+  - id: kubernetes-container-logs-${kubernetes.pod.uid}-${kubernetes.container.name}
```

Comment:
This creates an input per container, since we are going to get data duplication anyway, is there a more efficient way to do this with a single filestream input instead? Or fewer inputs?

Reply:
Thanks, resolving as out of scope for this change